scripts/populate.ts

```diff
@@ -0,0 +1,253 @@
+import type { User } from "../src/lib/types/User";
+import type { Assistant } from "../src/lib/types/Assistant";
+import type { Conversation } from "../src/lib/types/Conversation";
+import type { Settings } from "../src/lib/types/Settings";
+import { defaultEmbeddingModel } from "../src/lib/server/embeddingModels.ts";
+import { Message } from "../src/lib/types/Message.ts";
+
+import { addChildren } from "../src/lib/utils/tree/addChildren.ts";
+
+const rl = readline.createInterface({
+	input: process.stdin,
+	output: process.stdout,
+});
+
+rl.on("close", function () {
+	process.exit(0);
+});
+
+const possibleFlags = ["reset", "users", "settings", "assistants", "conversations"];
+const argv = minimist(process.argv.slice(2));
+const flags = argv["_"].filter((flag) => possibleFlags.includes(flag));
+
+async function generateMessages(preprompt?: string): Promise<Message[]> {
+	const isLinear = faker.datatype.boolean(0.5);
+	const isInterrupted = faker.datatype.boolean(0.05);
+
+	const messages: Message[] = [];
+
+	messages.push({
+		id: crypto.randomUUID(),
+		from: "system",
+		content: preprompt ?? "",
+		createdAt: faker.date.recent({ days: 30 }),
+		updatedAt: faker.date.recent({ days: 30 }),
+	});
+
+	let isUser = true;
+	let lastId = messages[0].id;
+	if (isLinear) {
+		const convLength = faker.number.int({ min: 1, max: 25 }) * 2; // must always be even
+
+		for (let i = 0; i < convLength; i++) {
+			lastId = addChildren(
+				{
+					messages,
+					rootMessageId: messages[0].id,
+				},
+				{
+					from: isUser ? "user" : "assistant",
+					content: faker.lorem.sentence({
+						min: 10,
+						max: isUser ? 50 : 200,
+					}),
+					createdAt: faker.date.recent({ days: 30 }),
+					updatedAt: faker.date.recent({ days: 30 }),
+					interrupted: i === convLength - 1 && isInterrupted,
+				},
+				lastId
+			);
+			isUser = !isUser;
+		}
+	} else {
+		const convLength = faker.number.int({ min: 2, max: 200 });
+
+		for (let i = 0; i < convLength; i++) {
+			addChildren(
+				{
+					messages,
+					rootMessageId: messages[0].id,
+				},
+				{
+					from: isUser ? "user" : "assistant",
+					content: faker.lorem.sentence({
+						min: 10,
+						max: isUser ? 50 : 200,
+					}),
+					createdAt: faker.date.recent({ days: 30 }),
+					updatedAt: faker.date.recent({ days: 30 }),
+					interrupted: i === convLength - 1 && isInterrupted,
+				},
+				faker.helpers.arrayElement([
+					messages[0].id,
+					...messages.filter((m) => m.from === (isUser ? "assistant" : "user")).map((m) => m.id),
+				])
+			);
+
+			isUser = !isUser;
+		}
+	}
+	return messages;
+}
+
+async function seed() {
+	console.log("Seeding...");
+	const modelIds = models.map((model) => model.id);
+
+	if (flags.includes("reset")) {
+		await collections.users.deleteMany({});
+		await collections.settings.deleteMany({});
+		await collections.assistants.deleteMany({});
+		await collections.conversations.deleteMany({});
+		console.log("Reset done");
```

Comment:
we should have similar `console.log("Reset started...");` and console statements for other types as well: `users`, `assistants` (like `console.log("Users created. Added 100 new users");`)